assets/node-exporter/daemonset.yaml

```diff
@@ -32,9 +32,10 @@ spec:
       automountServiceAccountToken: true
       containers:
       - args:
-        - --web.listen-address=127.0.0.1:9100
```

Comment:
I agree, using the same port on different interfaces can confusing, plus a pod has plenty of them